packages/client/hmi-client/src/page/project/components/tera-resource-sidebar.vue

```diff
@@ -159,4 +178,17 @@ nav {
 	overflow: hidden;
 	white-space: nowrap;
 }
+
+.p-tree:deep(.p-treenode-content:has(span[active='true'])),
```

Comment:
Why the `:has()`? did a simple `.p-treenode-content [active=true]` did not work?
Or is that you want the `.p-treenode-content` to highlight when the `span[active=true]` exist?

Reply:
I shortened the second rule but I still have to use the span. I use the span to trigger if it's active (you can see my custom template in the tree widget). Once span triggers I use the has to change the parent style